server/googleapi/googleCoordinates.js

```diff
@@ -0,0 +1,34 @@
+require('dotenv').config()
+const axios = require('axios')
+
+/**
+ Retrieves the coordinates of a stage location using Google Places API.
+ @param {string} destination - The destination of the stage.
+ @param {string} stageLocationName - The name of the stage location.
+ @returns {object} The coordinates of the stage location as an object { lat: number, lng: number }
+ @throws {Error} If there is an error while fetching place details.
+ */
+async function getStageCoordinates(destination, stageLocationName) {
```

Comment:
stage object? Lets keep it consistent to openai changes so theres no ambiguity in usage.

Reply:
so right now it returns an object as follows for example: {
  lat: 10.0,
  lng: 50.0
}